src/ia-book-actions.js

```diff
@@ -669,7 +693,21 @@ export default class IABookActions extends LitElement {
       secondsLeft = Math.round(secondsLeft); // round number
 
       // re-sync timer if gone off because of background window
-      await this.reSyncTimerIfGoneOff(secondsLeft);
+      // side effect: updates this.lendingStatus & kicks off lifecycle,
+      // if really updated, escape from here
+      const resync = await this.reSyncTimerIfGoneOff(secondsLeft);
```

Comment:
i updated `reSyncTimerIfGoneOff` to return boolean, returns true only IF resync had to happen.

this is also tied to `this.lendingStatus` lifecycle - and it was a potential chasm between lifecycle updates.  this boolean value clears up ambiguity.

line 707 below returns early if resync DID happen.